src/scripts/main.js

```diff
@@ -354,7 +354,46 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
```

Comment:
```suggestion
const table = document.querySelector('.dashboard tbody');
```
all rows should be inside tbody
![Screenshot 2024-09-21 at 18 37 55](https://github.com/user-attachments/assets/23025e78-4e9e-48ea-ab7e-3b358b291668)